patternfly/patternfly-elements
# docs: update installation / get started

## What I did

1.  Updated the "Getting Started" and "Quick Docs" pages to provide updated examples and some additional wording to make information clearer.
2. Added an "Installation" section to the component docs that mimics RHDS which gives them the install instructions for patternfly/elements and how to import that specific component.  

## Testing Instructions

1.  Please check out the deploy preview and check over the docs for information / wording.
